tests/unit-tests/test-class-quiz.php

```diff
@@ -1899,13 +1902,52 @@ public function testMaybeCreateQuizProgress_QuizWasAvailable_DoesntCreateQuizPro
 				],
 			]
 		);
-		Sensei_Utils::user_start_lesson( $user_id, $lesson_id );
```

Comment:
For some reason `Sensei_Utils::user_start_lesson` didn't work for me. I had to use the course enrolment directly.

Reply:
It didn't work but it looked like it was working because the tests were checking the comments-based progress.